entries/models.py

```diff
@@ -1,9 +1,11 @@
 from django.db import models
 from django.utils import timezone
-
+from django.conf import settings
+from datetime import datetime, timedelta
```

Comment:
I don't see you using `datetime`.

General hint: only use `import datetime` to prevent confusion with `from datetime import datetime` and then use `datetime.timedelta()` where needed. It's a bit more verbose but a common problem when writing code that you need to figure out if `datetime` is the package or the module. :wink: